internal/docstore/drivertest/drivertest.go

```diff
@@ -30,6 +30,9 @@ import (
 	"gocloud.dev/internal/docstore/driver"
 )
 
+// TODO(jba): Test RunActions with unordered=true. We can't actually test the ordering,
+// but we can check that all actions are executed even if some fail.
```

Comment:
Do we also have a test that does an ordered list of actions, with an expected error in the middle, and ensures it gets the right error back and that the actions after the error aren't executed?

And one that verifies that a malformed action is returned as an error and none of the actions are executed?

(If no to either please add as TODOs).

Reply:
Added TODOs.